pages/dkp/konvoy/2.2/release-notes/index.md

```diff
@@ -71,39 +71,39 @@ This new predictive analytics tool provides greater support productivity, speed,
-| velero | 3.2.0 | - chart: 3.2.0<br>- velero: 1.5.2 |
+| Application Name     | Platform Application                           | Version | Component Versions                                                                                                     |
+|----------------------|------------------------------------------------|---------|------------------------------------------------------------------------------------------------------------------------|
+|                      | cert-manager                                   | 1.7.1   | - chart: 1.7.1 - cert-manager: 1.7.1                                                                                   |
+| Dex                  | dex                                            | 2.9.14  | - chart: 2.9.14 - dex: 2.22.0                                                                                          |
+|                      | external-dns                                   | 6.1.8   | - chart: 6.1.8 - external-dns: 0.10.2                                                                                  |
+| Fluent Bit           | fluent-bit                                     | 0.19.20 | - chart: 0.19.20 - fluent-bit: 1.8.13                                                                                  |
+| Gatekeeper           | gatekeeper                                     | 3.7.0   | - chart: 3.7.0 - gatekeeper: 3.7.0                                                                                     |
+| Grafana              | grafana-logging                                | 6.22.0  | - chart: 6.22.0 - grafana: 8.3.6                                                                                       |
+| Loki                 | grafana-loki                                   | 0.33.2  | - chart: 0.33.1 - loki: 2.2.1                                                                                          |
+| Istio                | istio                                          | 1.11.6  | - chart: 1.11.6 - istio: 1.11.5                                                                                        |
+| Jaeger               | jaeger                                         | 2.29.0  | - chart: 2.29.0 - jaeger: 1.31.0                                                                                       |
+| Karma                | karma                                          | 2.0.1   | - chart: 2.0.1 - karma: 0.88                                                                                           |
+| Kiali                | kiali                                          | 1.47.0  | - chart: 1.47.0 - kiali: 1.47.0                                                                                        |
+| KNative              | knative                                        | 0.3.9   | - chart: 0.3.9 - knative: 0.22.3                                                                                       |
```

Comment:
I have seen this way of writing it several times, but when I google the app, it seems like it is written with lower-case?
```suggestion
| KNative              | knative                                        | 0.3.9   | - chart: 0.3.9 - knative: 0.22.3                                                                                       |
```